lib/validateMessage.js

```diff
@@ -63,6 +63,21 @@ exports.validateMessage = function validateMessage(raw) {
     var type = match[3];
     var scope = match[4];
     var subject = match[5];
+    config.scopes = config.scopes || {};
+    var validateScopes = config.scopes.validate || false;
+    var multipleScopesAllowed = config.scopes.multiple || false;
+    var allowedScopes = config.scopes.allowed || '*';
+    var scopeRequired = config.scopes.required || false;
+    var scopes = scope ? scope.split(',') : [];
+    var validateScope = (item) => {
```

Comment:
Refactoring this is pretty easy. I just need to convert it to a standard `for` loop instead of using `forEach` on the array. Was just trying to keep it as concise as possible.

+1 to finding a way to split things up. However, I'd rather we discuss that in its own issue and PR to implement (which I'm happy working on.) That way this PR is as concise as possible to the addition being added.